products/android/13.0/installing-custom-packages.php

```diff
@@ -63,29 +63,39 @@
   Older versions of Android grant Storage permissions at app installation time, so those users can skip this step.
 </p>
 <p>
-  Step 4) Keyman for Android will parse the metadata in the package. If the keyboard package includes a "welcome.htm"
+  Step 4) Keyman for Android will parse the metadata in the package. If the keyboard package includes a "readme.htm"
```

Comment:
This language is probably too technical! 
```suggestion
  Step 4) Keyman for Android will verify the package, and display more information about it, including package version and 'readme' notes if available.
```